pkg/kubeapiserver/options/authentication_test.go

```diff
@@ -747,7 +726,6 @@ jwt:
 
 	for _, testcase := range testCases {
 		t.Run(testcase.name, func(t *testing.T) {
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.AnonymousAuthConfigurableEndpoints, testcase.enableAnonymousEndpoints)
```

Comment:
revert the other changes in this file, we'll keep the feature disabled tests until emulation version no longer supports 1.33:

```
if !testcase.enableAnonymousEndpoints {
  featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33")) // last version the gate could be disabled
  featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.AnonymousAuthConfigurableEndpoints, testcase.enableAnonymousEndpoints)
}
```

Reply:
It doesn't let me disable the feature because I locked it to default in the config.

```
=== FAIL: pkg/kubeapiserver/options TestToAuthenticationConfig_Anonymous (0.05s)
    authentication_test.go:510: error setting StructuredAuthenticationConfiguration=true: cannot set feature gate AnonymousAuthConfigurableEndpoints to false, feature is locked to true
```

Reply:
calling `featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))` will unlock it and let you set it to false

Reply:
oops completely missed that line from the code snippet above. Thanks Ill make that change.